src/standalone/Snow/Snow.jl

```diff
@@ -293,66 +293,6 @@ Base.@kwdef struct SnowParameters{
     earth_param_set::PSE
 end
 
-"""
-   SnowParameters{FT}(Δt;
-                      density = MinimumDensityModel(200),
-                      z_0m = FT(0.0024),
-                      z_0b = FT(0.00024),
-                      α_snow = ConstantAlbedoModel(FT(0.8)),
-                      ϵ_snow = FT(0.99),
-                      θ_r = FT(0.08),
-                      Ksat = FT(1e-3),
-                      κ_ice = FT(2.21),
-                      ΔS = FT(0.1),
-                      scf = WuWuSnowCoverFractionModel(FT(0.106),FT(1.81), FT(0.08), FT(1.77), FT(1.0), FT(1),),
-                      earth_param_set::PSE) where {FT, PSE}
-
-An outer constructor for `SnowParameters` which supplies defaults for
-all arguments but `earth_param_set`.
-"""
-function SnowParameters{FT}(
```

Comment:
this constructor is used in a few more places that need to be updated:
- https://github.com/CliMA/ClimaLand.jl/blob/6e43b5c3c691bd21802441d88bb886a0b4ed5045/src/standalone/Snow/Snow.jl#L406
- https://github.com/CliMA/ClimaLand.jl/blob/6e43b5c3c691bd21802441d88bb886a0b4ed5045/src/standalone/Snow/Snow.jl#L492
- and a few tests